packages/app-info/README.md

```diff
@@ -0,0 +1,70 @@
+
+## @dotcom-reliability-kit/app-info
+
+A utility to get application information (e.g. the system code) in a consistent way. This module is part of [FT.com Reliability Kit](https://github.com/Financial-Times/dotcom-reliability-kit#readme).
+
+  * [Usage](#usage)
+    * [`appInfo.commitHash`](#appinfocommithash)
+    * [`appInfo.environment`](#appinfoenvironment)
+    * [`appInfo.region`](#appinforegion)
+    * [`appInfo.releaseDate`](#appinforeleasedate)
+    * [`appInfo.releaseVersion`](#appinforeleaseversion)
+    * [`appInfo.systemCode`](#appinfosystemcode)
+  * [Contributing](#contributing)
+  * [License](#license)
+
+
+## Usage
+
+Install `@dotcom-reliability-kit/app-info` as a dependency:
+
+```bash
+npm install --save @dotcom-reliability-kit/app-info
+```
+
+Include in your code:
+
+```js
+import appInfo from '@dotcom-reliability-kit/app-info';
+// or
+const appInfo = require('@dotcom-reliability-kit/app-info');
+```
+
+The `appInfo` object has several properties which can be used to access application information.
+
+### `appInfo.commitHash`
+
+Get the commit hash that the application last deployed. This will be a string (if `process.env.HEROKU_SLUG_COMMIT` is defined) or `null` otherwise. This relies on the [Heroku Dyno metadata labs feature](https://devcenter.heroku.com/articles/dyno-metadata) and will not be present in local development.
+
+### `appInfo.environment`
+
+Get the application environment, normally either `development` or `production`. This will be a string, using `process.env.NODE_ENV` and defaulting to `development`.
+
+### `appInfo.region`
+
+Get the application Heroku region. This will be a string (if `process.env.REGION` is defined) or `null` otherwise.
```

Comment:
Is it worth stating that this will not be present in local development or is that implicit?

Reply:
I _think_ it's implicit and not always true, because if a team sets a `REGION` environment variable in their `development` Vault config then it will be present?